.github/workflows/build.yml

```diff
@@ -26,7 +26,13 @@ jobs:
       MINGW_W64_VERSION: ${{steps.get-versions.outputs.MINGW_W64_VERSION}}
       PYTHON_VERSION_MINGW: ${{steps.get-versions.outputs.PYTHON_VERSION_MINGW}}
       TAG: ${{steps.get-tag.outputs.TAG}}
+      COMMIT_DATE_UNIX: ${{steps.get-tag.outputs.COMMIT_DATE_UNIX}}
+      BUILD_DATE: ${{steps.get-tag.outputs.BUILD_DATE}}
+      BUILD_DATE_UNIX: ${{steps.get-tag.outputs.BUILD_DATE_UNIX}}
     steps:
+      - uses: actions/checkout@v3
+        with:
+          sparse-checkout: .
```

Comment:
Why is `sparse-checkout` needed here? I guess it's nice for getting a smaller checkout if just wanting to inspect the git history, but the actual files within the llvm-mingw repo are quite minimal anyway?